### ISSUE TITLE
Cart context responses missing

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/forces-and-motion-basics#496]
For https://github.com/phetsims/qa/issues/1320

The context responses for the cart that were in the design doc and present in 2.6.0-dev.3 are missing in 2.6.0-rc.1. I don't see them in the a11y view or hear them with VO.

<img width="791" height="325" alt="Image" src="https://github.com/user-attachments/assets/ce15f6c3-9bf5-4ad0-bc37-2f193f1883dd" />

<img width="791" height="448" alt="Image" src="https://github.com/user-attachments/assets/74ad204a-fe59-4366-974e-4cf93c989a62" />

**Comment by samreid** [comment]
@jessegreenberg may I please request your help here? This issue was surprising since it was working in the dev test, and it is fixed if I put 

<details>

```diff

Subject: [PATCH] Fix focusability, see https://github.com/phetsims/forces-and-motion-basics/issues/477
---
Index: js/netforce/view/GoPauseButton.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/js/netforce/view/GoPauseButton.ts b/js/netforce/view/GoPauseButton.ts
--- a/js/netforce/view/GoPauseButton.ts	(revision b42173d739e155f8ba2c32d7af1558c0f9243986)
+++ b/js/netforce/view/GoPauseButton.ts	(date 1761674846740)
@@ -83,6 +83,8 @@
     // follow the precedent that description is added in the view.
     model.isRunningProperty.lazyLink( isRunning => {
 
+      console.log( isRunning )
+
       if ( isSettingPhetioStateProperty.value ) {
         return;
       }
@@ -112,15 +114,21 @@
           movementMessage = ForcesAndMotionBasicsFluent.a11y.netForceScreen.goPauseButton.cartMovingLeftStringProperty.value;
         }
 
-        this.addAccessibleContextResponse( movementMessage );
+        this.addAccessibleContextResponse( movementMessage, {
+          alertWhenNotDisplayed: true
+        } );
       }
       else {
         const position = model.cart.positionProperty.value;
         if ( Math.abs( position ) >= model.gameLength ) {
-          this.addAccessibleContextResponse( ForcesAndMotionBasicsFluent.a11y.netForceScreen.goPauseButton.cartStationaryStringProperty.value );
+          this.addAccessibleContextResponse( ForcesAndMotionBasicsFluent.a11y.netForceScreen.goPauseButton.cartStationaryStringProperty.value, {
+            alertWhenNotDisplayed: true
+          } );
         }
         else {
-          this.addAccessibleContextResponse( ForcesAndMotionBasicsFluent.a11y.netForceScreen.goPauseButton.cartPausedStringProperty.value );
+          this.addAccessibleContextResponse( ForcesAndMotionBasicsFluent.a11y.netForceScreen.goPauseButton.cartPausedStringProperty.value, {
+            alertWhenNotDisplayed: true
+          } );
         }
       }
     } );

```

</details>

What is surprising is that without `alertWhenNotDisplayed: true`, if I step through, I do not see any "shutoff valves" that would otherwise stop the message.
